pkg/machines/components/notificationArea.jsx

```diff
@@ -0,0 +1,86 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2017 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React, { PropTypes } from "react";
+import cockpit from 'cockpit';
+
+import { mouseClick } from "../helpers.es6";
+
+React;
+
+
+const Error = ({ description, message, action, actionMessage }) => {
+    const descriptionNode = description ? `${description} ` : null;
+    const messageNode = message ? (
+        <strong>
+            {`${message} `}
+        </strong>
+    ) : null;
+
+    const actionNode = action && actionMessage ?
+                       <a onClick={mouseClick(action)} className="alert-link">{actionMessage}</a> : null;
+    return (
+        <li>
+            {messageNode}
+            {descriptionNode}
+            {actionNode}
+        </li>
+    );
+};
+
+
+Error.propTypes = {
+    description: PropTypes.string.isRequired,
+    message: PropTypes.string.isRequired,
+    action: PropTypes.func.isRequired,
+    actionMessage: PropTypes.string.isRequired,
+};
+
+const NotificationArea = ({ errors, dismissable, onDismiss, id }) => {
```

Comment:
Can we reuse `Alert` from `inlineNotification.jsx`?

Reply:
Will be resolved after the final design decision